velox/exec/fuzzer/JoinFuzzer.cpp

```diff
@@ -299,6 +303,24 @@ class JoinFuzzer {
 
   VectorFuzzer vectorFuzzer_;
   std::unique_ptr<ReferenceQueryRunner> referenceQueryRunner_;
+
+  struct Stats {
+    // The number of iterations verified against reference DB.
+    size_t numVerified{0};
+
+    // The number of iterations that test cross product.
+    size_t numCrossProduct{0};
+
+    void print(size_t totalIterations) const {
```

Comment:
Let make totalIterations a member?
size_t numIterations;

And add a toString() method for stats
LOG(INFO) << stats_.toString()? 
Thanks!

Reply:
So I have tried making this change, but it turns out to be less readable then before. The original output is 
```
W0814 12:39:44.775323 847942 JoinFuzzer.cpp:1481] ==============================> Done with iteration 0
I0814 12:39:44.775427 847942 JoinFuzzer.cpp:318] Total iterations tested: 1
I0814 12:39:44.775455 847942 JoinFuzzer.cpp:319] Total iterations verified against reference DB: 1 (100.00%)
I0814 12:39:44.775501 847942 JoinFuzzer.cpp:321] Total iterations testing cross product: 0 (0.00%)
```

After changing the code to `LOG(INFO) << stats_.toString()`, the output becomes
```
W0814 12:36:05.199414 779680 JoinFuzzer.cpp:1485] ==============================> Done with iteration 3
I0814 12:36:05.199489 779680 JoinFuzzer.cpp:1491] Total iterations tested: 4
Total iterations verified against reference DB: 3 (75.00%)
Total iterations testing cross product: 0 (0.00%)
```